packages/manager/src/features/CloudPulse/Alerts/CreateAlert/GeneralInformation/ResourceMultiSelect.tsx

```diff
@@ -0,0 +1,95 @@
+import { useFormikContext } from 'formik';
+import * as React from 'react';
+
+import { Autocomplete } from 'src/components/Autocomplete/Autocomplete';
+import {
+  useLinodeResourcesQuery,
+  useResourcesQuery,
+} from 'src/queries/cloudpulse/resources';
+interface CloudPulseResourceSelectProps {
+  /**
+   * cluster type selected by the user
+   */
+  cluster: boolean;
+  /**
+   * name used for the component to set formik field
+   */
+  name: string;
+  /**
+   * region selected by the user
+   */
+  region: string | undefined;
+  /**
+   * service type selected by the user
+   */
+  serviceType: string | undefined;
+}
+
+export const CloudPulseMultiResourceSelect = (
+  props: CloudPulseResourceSelectProps
+) => {
+  const resourceOptions: any = {};
+  const { cluster, name, region, serviceType } = { ...props };
+  const formik = useFormikContext();
+  const [selectedResource, setResource] = React.useState<any>([]);
+  const filterResourcesByRegion = (resourcesList: any[]) => {
```

Comment:
please avoid using any instead create a type for it. This is the one we are using. Once you sync with latest develop branch, you can re-use the existing resource select component used in cloud pulse.

`
export interface CloudPulseResources {
  id: string;
  label: string;
  region?: string;
}
`